paasta_itests/setup_marathon_job.feature

```diff
@@ -18,6 +18,19 @@ Feature: setup_marathon_job can create a "complete" app
       And we run setup_marathon_job until it has 5 task(s)
      Then we should see the number of instances become 5
 
+  Scenario: marathon apps can be scaled up with at-risk hosts
+    Given a working paasta cluster
+      And a new healthy app to be deployed, with bounce strategy "crossover" and drain method "noop" and constraints [["hostname", "UNIQUE"]]
+      And an old app to be destroyed with constraints [["hostname", "UNIQUE"]]
+    When there are 2 old healthy tasks
+     And we mark the host "mesosslave" as at-risk
```

Comment:
There's no guarantee that there's gonna be a task on "mesosslave", since there are 2 instances and 3 slaves, right?

Can we have this step pick one of the hosts where this app is actually running?